crates/ruff_linter/src/rules/pylint/snapshots/ruff_linter__rules__pylint__tests__preview__PLC0206_dict_index_missing_items.py.snap

```diff
@@ -0,0 +1,215 @@
+---
+source: crates/ruff_linter/src/rules/pylint/mod.rs
+---
+--- Linter settings ---
+-linter.preview = disabled
++linter.preview = enabled
+
+--- Summary ---
+Removed: 7
+Added: 7
+
+--- Removed ---
+PLC0206 Extracting value from dictionary without calling `.items()`
+  --> dict_index_missing_items.py:9:1
+   |
+ 8 |   # Errors
+ 9 | / for instrument in ORCHESTRA:
```

Comment:
Is it necessary to make this a preview-only change? To suppress PLC0206 today, you have to put the noqa on the line where the `for` statement starts. Will this change with this PR?

The only example that I can think of is:

```py
ORCHESTRA = dict()

for (  # noqa PLC0206
    instrument
) in ORCHESTRA:
    print(f"{instrument}: {ORCHESTRA[instrument]}")


```

But we could fix this by using `parenthesized_range` for the for-target.